geomstats/_backend/pytorch/__init__.py

```diff
@@ -31,6 +31,7 @@
 from torch import mean, meshgrid, moveaxis, ones, ones_like, polygamma, quantile
 from torch import repeat_interleave as repeat
 from torch import reshape, stack, trapz, uint8, unique, vstack, zeros, zeros_like
+from torch.special import gammaln  # NOQA
```

Comment:
for consistency, let's import as `_gammaln` (for readability, everything  that is public here is assumed to be part of the backed).

Reply:
Good point, done.